sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -1199,6 +1200,39 @@ export class NotificationActionsService {
     );
   }
 
+  /**
+   * Create supporting user information notification for student.
+   * @param notification notification details.
+   * @param auditUserId user who sends the notification.
+   */
+  async saveSupportingUserInformationNotification(
```

Comment:
This method must receive entity manager to be executed within a transaction in 

[createSupportingUsers Workers](https://github.com/bcgov/SIMS/blob/1b5198f8fbc67ae045890a9491414a36a3451bff/sources/packages/backend/apps/workers/src/controllers/supporting-user/supporting-user.controller.ts#L77)